django/contrib/messages/storage/cookie.py

```diff
@@ -14,6 +14,10 @@ class MessageEncoder(json.JSONEncoder):
     """
     message_key = '__json_message'
 
+    def __init__(self, *args, **kwargs):
+        kwargs['separators'] = (',', ':')
```

Comment:
I we're aiming for the lowest storage size we use `ensure_ascii=False` as well

```suggestion
        kwargs.setdefault('separators', (',', ':'))
        kwargs.setdefault('ensure_ascii', False)
```

Reply:
But cookie values only allow ASCII :D Then again it might make more sense to completely change the encoding (also see https://code.djangoproject.com/ticket/32191#comment:6 which actually led me to the byte savings…)